File: src/main/java/com/dongfg/project/api/controller/admin/AdminController.java
package com.dongfg.project.api.controller.admin;

import com.dongfg.project.api.dto.CommonResponse;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

/**
 * @author dongfg
 * @date 18-1-24
 */
@RequestMapping("/admin")
@RestController
public class AdminController {

    @GetMapping
    public CommonResponse<String> index() {
        return CommonResponse.<String>builder().success(true).build();
    }
}

File: src/main/kotlin/com/dongfg/project/api/web/controller/admin/AdminController.kt
package com.dongfg.project.api.web.controller.admin

import com.dongfg.project.api.web.payload.GenericPayload
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RequestMapping("/admin")
@RestController
class AdminController {

    @GetMapping
    fun index(): GenericPayload {
        return GenericPayload(true)
    }

}